docs/en/FAQ/v9-version-upgrade.md

```diff
@@ -20,4 +20,5 @@ Notice **Incompatibility (1)**, the UI template configuration protocol is incomp
    These data would be incompatible with previous versions.
    Make sure to remove the older `ServiceTraffic` table before OAP(v9) starts. 
    OAP would generate the new table in the start procedure, and recreate all existing services when traffic comes.
+   Since V9.1, SQL Database: move `Tags list` from `Segment`,`Logs`,`Alarms` to their additional table, remove them before OAP starts.
```

Comment:
```suggestion
   Since V9.1, SQL Database: move `Tags list` from `Segment`, `Logs`, `Alarms` to their additional tables, remove them before OAP starts.
```

I think we don't have to remove these old tables? As you are adding a new table not change one(abandon some columns should be safe?

Reply:
Let's just fix English grammar issues.